### DISCUSSION TITLE
gcode preview

### DISCUSSION THREAD

**Discussion by jlillie3** [mriscoc/Ender3V2S1#8] [General]
Do these configurations contain the gcode preview that CHEP was referring to in his [recent video?](https://youtu.be/Q-OA9_t9bGc)? If so, how can I enable it when I compile the firmware?

**Comment by mriscoc** [comment]
No, some features are not yet enabled in the source code.

**Reply by hddvrn** [reply]
when can I expect?

**Reply by mriscoc** [reply]
New version released, g-code preview included.
